krun/scheduler.py

```diff
@@ -148,6 +156,18 @@ def update_num_mails_sent(self):
         self._reset()
         self._parse()  # update stats
 
+    def update_num_reboots(self):
+        """Updates the reboot count header in the manifest file."""
+
+        debug("Increment reboot count in manifest")
+        fh = self._open()
+        fh.seek(self.num_reboots_offset)
+        fh.write(ManifestManager.NUM_REBOOTS_FMT % (self.num_reboots + 1))
```

Comment:
Should there be an `assert` here that `num_reboots` can't overflow `NUM_REBOOTS_FMT`? Or does Python's stringf do that implicitly?